waiter/src/waiter/kv.clj

```diff
@@ -94,9 +95,9 @@
   given the underlying ZooKeeper implementation."
   [key]
   (when (re-matches #".*/.*" key)
-    (throw (ex-info "Key may not contain '/'" {:key key})))
+    (throw (ex-info "Key may not contain '/'" {:key key :ex-type ::invalid-zk-key})))
```

Comment:
Can we change this to `invlaid-key` so that users of this keyword don't have to know the exception was thrown from ZK?